src/Network.ts

```
@@ -2,17 +1,66 @@
       chain_name: chain.name,
+      prettyName: chain.name,
+      name: chain.name,
       authzAminoSupport: chain.params.authz ?? false,
       prefix: chain.bech32_prefix,
       txTimeout,
       coinType: chain.slip44,
       chainId: chain.chain_id,
-      restUrl: restUrl ?? new CosmosDirectory().restUrl(chain.name)
+      rpcUrl: rpcUrl ?? new CosmosDirectory().rpcUrl(chain.name),
+      restUrl: restUrl ?? new CosmosDirectory().restUrl(chain.name),
+      gasPriceStep: this.getGasPriceStep(chain),
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)